dali/base/dadfs.cpp

```diff
@@ -4312,11 +4312,19 @@ protected: friend class CDistributedFilePart;
         StringBuffer fullname;
         CIArrayOf<CIStringArray> newNames;
         unsigned i;
-        for (i=0;i<width;i++) {
+        for (i=0;i<width;i++)
+        {
             newNames.append(*new CIStringArray);
             CDistributedFilePart &part = parts.item(i);
-            for (unsigned copy=0; copy<part.numCopies(); copy++) {
-                makePhysicalPartName(newname, i+1, width, newPath.clear(), 0, os, myBase, hasDirPerPart());
+            for (unsigned copy=0; copy<part.numCopies(); copy++)
+            {
+                unsigned cn = copyClusterNum(i, copy, nullptr);
+                unsigned numStripedDevices = queryPartDiskMapping(cn).numStripedDevices;
+                unsigned stripeNum = 0;
+                if (numStripedDevices>1)
+                    stripeNum = (i%numStripedDevices)+1;
```

Comment:
more: also hash of logical filename/base hash stored in the file descriptor?

Reply:
I deliberately left this functionality off in this draft so far (but there is a comment somewhere), so prove the rest worked.